lib/getStatistics.js

```diff
@@ -97,11 +118,78 @@ function getNumberOfProperties(object) {
     return Object.keys(object).length;
 }
 
+function getDrawCallStatisticsForNode(gltf, nodeId) {
```

Comment:
`getDrawCallStatisticsForNode` and  `getDrawCallStatistics` are similar enough that they should share a helper function that gets the statistics for an array of primitives.